packages/design-system/token-transformer/index.ts

```diff
@@ -0,0 +1,50 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { default as convertFile } from './src/lib';
+
+let inputPath: path.ParsedPath | undefined, outputPath: path.ParsedPath | undefined;
+
+// Set up input and output paths
+process.argv.forEach((val, index) => {
+  if (index === 2) { 
+    inputPath = path.parse(val);
+  }  
+  if (index === 3) {
+    outputPath = path.parse(val);
+  }
+});
+
+if (inputPath === undefined) {
```

Comment:
Absolutely. If we're planning on making this more of an built-out tool rather than a single one-off manual script, then I'll clean up and refactor. Since I was using it as a single file script, I wanted to avoid dependencies, but a library is a different case.

Reply:
See #302.